proarc-common/src/main/java/cz/cas/lib/proarc/common/mods/ndk/NdkMapper.java

```diff
@@ -148,8 +149,13 @@ public ModsDefinition fromJsonObject(ObjectMapper jsMapper, String json, Context
             } else {
                 descriptionStandard.setValue(ModsConstants.VALUE_DESCRIPTIONSTANDARD_AACR);
             }
-            mods.getRecordInfo().get(0).getDescriptionStandard().add(0, descriptionStandard);
-            return mods;
+            if (mods.getRecordInfo().size() == 0) {
+                RecordInfoDefinition ri = new RecordInfoDefinition();
+                ri.getDescriptionStandard().add(0, descriptionStandard);
+                mods.getRecordInfo().add(0, ri);
+            } else {
+                mods.getRecordInfo().get(0).getDescriptionStandard().add(0, descriptionStandard);
```

Comment:
Nepřidá `.add(0, descriptionStandard)` při každém update nový element?

Reply:
Nepřidá, je to otestováno. Vždy se zachovává jen jen poslední záznam. Původní záznam se přepíše.